README.md

```diff
@@ -9,26 +9,61 @@ it on [Google Play](https://play.google.com/store/apps/details?id=org.wordpress.
 ## Build Instructions ##
 
 1. Make sure you've installed [JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and [Android Studio](https://developer.android.com/studio/index.html), a _Standard Setup_ would work.
-2. Clone this GitHub repository.
-3. Copy `gradle.properties-example` to `gradle.properties`.
-4. In Android Studio open the project from the local repository as a **Gradle project** (this will auto-generate `local.properties` with the SDK location).
-5. Make sure you have an emulation device setup in AVD Manager (_Tools → Android → AVD Manager_).
-6. Run.
+1. `git clone git@github.com:wordpress-mobile/WordPress-Android.git` in the folder of your preference.
+1. `cd WordPress-Android` to enter the working directory.
+1. `cp gradle.properties-example gradle.properties` to copy the example credentials file to your home folder.
```

Comment:
This command won't copy anything to the home folder 🤔 

Perhaps:

>to set up the sample app credentials file.

Reply:
Aha 🤔 yes, thanks for catching that!